.github/workflows/ci.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           - name: Reuse ccache directory
             uses: actions/cache@v4
             with:
-                path: ~/.ccache
+                path: ~/.cache/ccache
```

Comment:
I noticed that it broke CI for Ubuntu 20.04 (it started to fail in post reuse ccache)

I suppose that in Ubuntu 20.04 correct path is `~/.ccache` and for Ubuntu 22.04 and 24.04 correct path is `~/.cache/ccache`

Probably, you can define a ccache directory in matrix and use the path here

Reply:
I'll fix here: https://github.com/userver-framework/userver/pull/754